### DISCUSSION TITLE
Best practice for getting local setup from a live client site that uses Understrap

### DISCUSSION THREAD

**Discussion by EricJammin** [holger1411/understrap#2219] [Q&A]
I am working with a new client, and although we will eventually be building a new website from scratch, we are first making some updates to their existing site, which is using an older version of Understrap theme. Unfortunately the theme files have been modified and added to, rather than using a child theme.

To keep this short, I'm basically just looking for some direction on the best approach to take with regards to npm and the build process, so that I can make CSS changes as needed and then upload the newly compiled files to the server.

I have already downloaded the theme files to my local folder. Do I just need to run npm install within the local theme folder from my terminal, or will there be additional things I need to configure? I can see the current files are still running bootstrap v4, FWIW. Any guidance is greatly appreciated!

**Comment by bacoords** [comment]
Can you check the package.json or theme and let us know what version? Older versions used a Gulp build process and newer versions use Rollup, so that's one big difference. 

You should be able to run `nvm use` to make sure you have the right node version running before you run `npm install`. If there's no `.nvmrc` file in the theme, then we know it's a really old version!
